app/qml/RegistrationForm.qml

```diff
@@ -161,138 +162,39 @@ Rectangle {
       anchors.horizontalCenter: parent.horizontalCenter
     }
 
-    Row {
+    PasswordField {
+      id: passwordField
       width: registerForm.width
       height: fieldHeight
-      spacing: 0
-
-      Rectangle {
-        id: iconContainer2
-        height: fieldHeight
-        width: fieldHeight
-        color: root.bgColor
-
-        Image {
-          anchors.margins: (fieldHeight / 4)
-          id: icon2
-          height: fieldHeight
-          width: fieldHeight
-          anchors.fill: parent
-          source: 'lock.svg'
-          sourceSize.width: width
-          sourceSize.height: height
-          fillMode: Image.PreserveAspectFit
-
-          MouseArea {
-            anchors.fill: parent
-            onClicked: {
-              if (password.echoMode === TextInput.Normal) {
-                password.echoMode = TextInput.Password
-              } else {
-                password.echoMode = TextInput.Normal
-              }
-            }
-          }
-        }
-
-        ColorOverlay {
-          anchors.fill: icon2
-          source: icon2
-          color: root.fontColor
-        }
-      }
-
-      TextField {
-        id: password
-        width: parent.width - iconContainer.width
-        height: fieldHeight
-        font.pixelSize: InputStyle.fontPixelSizeNormal
-        color: root.fontColor
-        placeholderText: qsTr("Password")
-        echoMode: TextInput.Password
-        inputMethodHints: Qt.ImhNoPredictiveText | Qt.ImhNoAutoUppercase
-        font.capitalization: Font.MixedCase
-
-        background: Rectangle {
-          color: root.bgColor
-        }
-      }
+      fontColor: root.fontColor
+      bgColor: root.bgColor
     }
 
     Rectangle {
       id: passBorder
       color: InputStyle.panelBackgroundDark
       height: 2 * QgsQuick.Utils.dp
-      y: password.height - height
-      opacity: password.focus ? 1 : 0.6
+      y: fieldHeight - height
+      opacity: passwordField.password.focus ? 1 : 0.6
       width: registerForm.width - fieldHeight / 2
       anchors.horizontalCenter: parent.horizontalCenter
     }
 
-    Row {
+    PasswordField {
+      id: passwordConfirmField
```

Comment:
We should probably change placeholder text to "Repeat password" or something similar so that there are not two times fields with text "Password"

Reply:
It was "Confirm password" before